src/core/TriangleSplitter.js

```diff
@@ -7,13 +7,23 @@ import { isTriDegenerate } from './utils/triangleUtils.js';
 const EPSILON = 1e-10;
 const COPLANAR_EPSILON = 1e-10;
 const PARALLEL_EPSILON = 1e-10;
+const COPLANAR_EPSILON_MAX = 1e-4;
+//*/
+
+// XAVIER: increase epsilon values:
+/*const EPSILON = 1e-5;
+const COPLANAR_EPSILON = 1e-5;
+const PARALLEL_EPSILON = 1e-5; //*/
+
 const _edge = new Line3();
 const _foundEdge = new Line3();
 const _vec = new Vector3();
 const _triangleNormal = new Vector3();
 const _planeNormal = new Vector3();
 const _plane = new Plane();
 const _splittingTriangle = new ExtendedTriangle();
+const _planeCenter = new Vector3();
+let _minEdgeSize = Infinity;
```

Comment:
Since `_minEdgeSize` is a float there's no reason to make this global. The rest of the values are global to avoid allocation memory for new objects like Vector3 unnecessarily.